Sources/AppearanceManager.swift

```diff
@@ -37,9 +37,14 @@ class AppearanceManager: NSObject {
         // For the edit selection indicators
         UITableView.appearance().tintColor = theme.colors.orangeUI
         UISegmentedControl.appearance().tintColor = theme.colors.orangeUI
+        
         UISwitch.appearance().onTintColor = theme.colors.orangeUI
-        UISearchBar.appearance().barTintColor = .white
+        UISwitch.appearance().thumbTintColor = theme == PresentationTheme.darkTheme ? theme.colors.background : nil
+        UISwitch.appearance().tintColor = theme.colors.switchBackgroundColor
+        UISwitch.appearance().backgroundColor = theme.colors.switchBackgroundColor
+        UISwitch.appearance().cornerRadius = 16
```

Comment:
I really don't think we need the cornerRadius

Reply:
If the cornerRadius is not set, the backgroundColor leaks outside the switch. The product looks something like this 
<img width="50" alt="untitled" src="https://user-images.githubusercontent.com/5487597/41529637-c1299aae-7328-11e8-8862-a1629233ee39.png">

Reply:
either we use it wrong or it's a UIKit bug and we should file a radar with a simple sample project that shows the issue. setting the corner radius could bite us one day if Apple changes the control from what it is right now

Reply:
just did a quick search on Google and a lot run into that so leave it for now, file a radar and add a comment